go-controller/pkg/types/const.go

```diff
@@ -108,7 +108,8 @@ const (
 	OvnHostToSvcOFTopoVersion      = 3
 	OvnPortBindingTopoVersion      = 4
 	OvnRoutingViaHostTopoVersion   = 5
-	OvnCurrentTopologyVersion      = OvnRoutingViaHostTopoVersion
+	OvnL4ACLConsolidatedVersion    = 6
+	OvnCurrentTopologyVersion      = OvnL4ACLConsolidatedVersion
```

Comment:
based on upstream meeting discussion: let's remove the topology version bump and use the same logic as egress firewall.